tests/fixture_tailwind_remote_classes/nestedPlugins/nested/nested/nested/veryNestedPlugin.ts

```diff
@@ -0,0 +1,19 @@
+import type { Plugin } from "../../../../../../src/server/types.ts";
+import VeryNestedPluginComponent from "../../../../components/VeryNestedPluginComponent.tsx";
+
+const currentUrl = new URL(import.meta.url);
+currentUrl.pathname = currentUrl.pathname.split("/").slice(0, -5).join("/") +
+  "/";
+
+export const veryNestedPlugin = {
+  name: "very nested plugin",
+  location: import.meta.url,
+  projectLocation: currentUrl.href,
+  routes: [{
+    path: "routeFromVeryNestedPlugin",
+    handler: (_req, ctx) => {
+      return ctx.render();
+    },
+    component: VeryNestedPluginComponent,
```

Comment:
Am I right in saying the `handler` property isn't needed if `component` is defined? Ditto for other plugins for this fixture.

Reply:
You're right -- thanks.